psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -270,13 +270,16 @@ def scf_iterate(self, e_conv=None, d_conv=None):
 
     # does the JK algorithm use severe screening approximations for early SCF iterations?
     early_screening = self.jk().get_early_screening()
+    # maximum number of scf iterations to run after early screening is disabled
+    scf_maxiter_post_screening = core.get_option('SCF', 'COSX_MAXITER_FINAL')
```

Comment:
Is there any practical difference between cosx_maxiter_final == -1 and -10? If the total total limit is still controlled by SCF_MAXITER, then no? If that's the case, maybe only allow -1.